internal/node/process_mgr.go

```diff
@@ -0,0 +1,17 @@
+package nexnode
+
+import agentapi "github.com/synadia-io/nex/internal/agent-api"
+
+// TODO: this will be utilized in a forthcoming PR
+type ProcessManager interface {
```

Comment:
Is this meant to be what I was calling `WorkloadManager`?

Reply:
I strongly suggest we call this `WorkloadManager` -- it does indeed seem to be that. I like this interface very much.